lib/screens/event/event.dart

```diff
@@ -107,6 +108,16 @@ class _EventPageState extends State<EventPage> {
     update();
   }
 
+  //Bör denna vara async som de andra funktionerna?
+  void goToSettings() {
+    Navigator.push(
+            context, MaterialPageRoute(builder: (context) => SettingsPage()))
+        .then((_) {
+      //uppdaterar sidan så man ser sin ändring
+      initState();
```

Comment:
I think this should be a call to update instead of initState, since I think the event page state is still loaded and not completelty rebuilt when we return to it.